regression-test/suites/delete_p0/test_delete.groovy

```diff
@@ -483,4 +483,33 @@ suite("test_delete") {
     sql "set experimental_enable_nereids_planner = false;"
     sql "set @data_batch_num='2024-01-31 00:00:00';"
     sql "delete  from bi_acti_per_period_plan where data_batch_num =@data_batch_num; "
+
+    // delete bitmap
+    sql "drop table if exists table_bitmap"
+    sql """
```

Comment:
And also should not support array,map,struct

Reply:
already ban